lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss

```diff
@@ -1,13 +1,12 @@
-.adf-task-form {
+    .adf-task-form {
```

Comment:
Indentation looks wrong in this file after your changes

Reply:
Done